Lib/asyncio/staggered.py

```diff
@@ -12,7 +12,8 @@
 class _Done(Exception):
     pass
 
-async def staggered_race(coro_fns, delay, *, loop=None):
+
+async def staggered_race(coro_fns, delay):
```

Comment:
Let's keep the loop kwarg and deprecate it

Reply:
I did this with `warnings._deprecated` -- is that right? It might be better if we just stick to `warnings.warn` here.

Reply:
Deprecation is not needed, it is private API